frontend/src/components/History/index.jsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 import ReactTable from "react-table-6";
```

Comment:
I'm still seeing unexpected (incorrect) results for trying to filter this table by a column (sorting by num comments, 7 is somehow between 67 and 70 on the 2nd page):
![Screen Shot 2023-03-14 at 5 04 11 PM](https://user-images.githubusercontent.com/85561396/225136183-5578c367-9110-4744-b222-251cdf78afab.png)

Reply:
Oh that's because it's searching for items containing 7. Should it be starts-with instead?